python/lambda-from-container/README.md

```diff
@@ -0,0 +1,79 @@
+Lambda Function from Container
+=============================
+
+* This example will produce a **Lambda function** from a **Container Image**.
+* The **Container Image** is defined in `app.py`. 
+  * If `use_pre_existing_image` is **True**, then it is expected that a **Container Image** was **created elsewhere** and will be used by this CDK code.
+  * If `use_pre_existing_image` is **False**, then a new container will be created using the `Dockerfile` from the the `lambda-image` sub-directory.
```

Comment:
Since this uses assets, please add a note about `cdk bootstrap`

Reply:
`ecr_assets` wasn't being used so I removed the reference to it.